doc/rule-descriptions.md

```diff
@@ -12,62 +12,62 @@
-| [server-side-image-map](https://dequeuniversity.com/rules/axe/4.6/server-side-image-map?application=RuleDescription)             | Ensures that server-side image maps are not used                                                                                                      | Minor                    | cat.text-alternatives, wcag2a, wcag211, section508, section508.22.f               | needs&nbsp;review          |                                                                                                                                                                                                                                                                    |
-| [svg-img-alt](https://dequeuniversity.com/rules/axe/4.6/svg-img-alt?application=RuleDescription)                                 | Ensures &lt;svg&gt; elements with an img, graphics-document or graphics-symbol role have an accessible text                                           | Serious                  | cat.text-alternatives, wcag2a, wcag111, section508, section508.22.a, ACT          | failure, needs&nbsp;review | [7d6734](https://act-rules.github.io/rules/7d6734)                                                                                                                                                                                                                 |
-| [td-headers-attr](https://dequeuniversity.com/rules/axe/4.6/td-headers-attr?application=RuleDescription)                         | Ensure that each cell in a table that uses the headers attribute refers only to other cells in that table                                             | Serious                  | cat.tables, wcag2a, wcag131, section508, section508.22.g                          | failure, needs&nbsp;review | [a25f45](https://act-rules.github.io/rules/a25f45)                                                                                                                                                                                                                 |
-| [th-has-data-cells](https://dequeuniversity.com/rules/axe/4.6/th-has-data-cells?application=RuleDescription)                     | Ensure that &lt;th&gt; elements and elements with role=columnheader/rowheader have data cells they describe                                           | Serious                  | cat.tables, wcag2a, wcag131, section508, section508.22.g                          | failure, needs&nbsp;review | [d0f69e](https://act-rules.github.io/rules/d0f69e)                                                                                                                                                                                                                 |
```

Comment:
We could put 14.b on these I think. It's not exactly, but it's meant to test the same thing.